<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpawnParameterEnemyMineWidget</class>
 <widget class="QWidget" name="SpawnParameterEnemyMineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="passiveMineButton">
       <property name="text">
        <string>Passive mine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="droppedMineButton">
       <property name="text">
        <string>Dropped mine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mineDropperButton">
       <property name="text">
        <string>Mine dropper</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="mineDropperSettingsWidget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Orientation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="orientationComboBox">
          <item>
           <property name="text">
            <string>0: Top-left to bottom-right</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1: Horizontal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2: Bottom-left to top-right</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3: Top-left to bottom-right</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Metatile coverage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="metatileCoverageSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>15</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
